Makefile

```
@@ -107,8 +103,7 @@ test: venv
 
 ## spellcheck     to check spelling
 .PHONY: spellcheck
-spellcheck: venv
-	$(call ensure_package,pre_commit)
+spellcheck: _ensure-pre-commit
 	$(VENVDIR)/bin/python3 -m pre_commit run --all-files --hook-stage manual codespell
```

Would it be better to factor out the common pre-commit invocation to something like

```make
PRECOMMIT = $(VENVDIR)/bin/python3 -m pre_commit run --all-files
```

And then call `lint` as `$(PRECOMMIT)` and `spellcheck` as

```suggestion
	$(PRECOMMIT) --hook-stage--hook-stage manual codespell
```

Or is it worth just leaving for consistency with the other makefiles?

---

> consistency with the other makefiles

I think this is the main goal here (& if we used `PRECOMMIT`, it would be really just for the abstraction---there's no reasonable use for the Makefile variable)